docs/pages/zenroom-javascript3.md

```diff
@@ -12,99 +12,216 @@ Now let's add some interactivity and see how we can play and interact with Zenco
 
 Let’s start by creating a standard React project with the [CRA](https://reactjs.org/docs/create-a-new-react-app.html) tool, and add Zenroom as a dependency
 
+<!--- tabs:start -->
+
+### **npm**
+
+```bash
+npx create-next-app@latest zenroom-react-test \
+    --js \
+    --src-dir \
+    --disable-git \
+    --app \
+    --use-npm \
+    --no-eslint \
+    --no-tailwind \
+    --no-turbopack \
+    --no-import-alias
+```
```

Comment:
I have mixed feelings about this, I know that with next.js everything goes smooth, but I really wanted to avoid to use a framework of a framework... that's why I went with a plain react app... maybe we can do an article on how to use it with different more famous js/ts frameworks also angular/svelte/next/vue, BUT keeping at least one very simple example with plain react...

Reply:
ok, I just follow the info in the [react documentation](https://react.dev/learn/start-a-new-react-project). Then will try to use plain react

Reply:
Done with plain react and save the `next` documentation in a different file